src/assets/global.css

```diff
@@ -1,21 +1,24 @@
-html {
-  font-family: Arial, Helvetica, sans-serif;
-  background: white;
-  perspective: 1000px;
-  height: 100%;
-  width: 100%;
-}
-
-body {
-  height: 100%;
-  width: 100%;
-  margin: 0;
-}
-
-#app {
-  height: 100%;
-  left: 0;
-  position: absolute;
-  right: 0;
-  top: 0;
-}
\ No newline at end of file
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&display=swap");
```

Comment:
Already checked out the licence? Also document in our [third-party-license-notice](https://github.com/Gamify-IT/third-party-license-notice) repo. And did you already checked out if there is a npm package for this, so we can easily install it with npm?

Reply:
- [ ] Please download the font and include it from the file system, as the method used is illegal under German law: https://www.ihk.de/konstanz/recht-und-steuern/handel-wettbewerb/reinfo/abmahnungen-wegen-google-fonts-5641026
- [ ] Presumably it is a SIL Open Font License nevertheless this has to be checked and added in the third-party-license-notice repo